src/client/driver/native-dialog-tracker/index.js

```diff
@@ -54,18 +57,22 @@ export default class NativeDialogTracker {
         this.contextStorage.setItem(ERROR_IN_HANDLER, dialog);
     }
 
+    static _getPageUrl () {
+        return eval(gettingPageUrlProcessedScript);
```

Comment:
what `eval` is it? it is `window.eval` here?
I suppose we should use `eval` from hammerhead because `window.eval` can be overridden by user script